spec/factories/vacancies.rb

```diff
@@ -7,6 +7,9 @@
     enable_job_applications { true }
     benefits { Faker::Lorem.paragraph(sentence_count: 4) }
     completed_step { 7 }
+    completed_steps do
```

Comment:
in this file we have 

```
    trait :draft do
      status { :draft }
    end
```

You mat want to override that field for that trait

Reply:
It's valid for a draft vacancy to have 7 completed steps (all steps except review). If the only difference between the trait `:draft` and the basic factory vacancy is the status, then I think it should have all the steps completed (all the attributes are set).